provider/elastictranscoder/aws.go

```diff
@@ -285,9 +285,18 @@ func (p *awsProvider) JobStatus(job *db.Job) (*provider.JobStatus, error) {
 			Width:    aws.Int64Value(resp.Job.Input.DetectedProperties.Width),
 		}
 	}
+	statusMessage := ""
+	if len(resp.Job.Outputs) > 0 {
+		statusMessage = aws.StringValue(resp.Job.Outputs[0].StatusDetail)
+		if strings.Contains(statusMessage, ":") {
+			errorMessage := strings.Split(statusMessage, ":")[1]
```

Comment:
I wonder if there's a risk of having a ``:`` in the statusMessage, in which case we should probably use ``SplitN(statusMessage, ":", 2)``.